### ISSUE TITLE
Ensure ARIA roles, states, and properties are valid

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#63]
## Description
When ARIA attributes (state, roles, and properties) are not used correctly assistive technology may not correctly function as expected. Valid markup should always be used.

### SlideBlock
- [x] The SlideBlock control is inaccessible for the screen reader user using the Tab key as the same content (i.e active slide) is announced again and again making the information redundant. The value of tabindex and the aria-hidden attribute is not updated when using the keyboard TAB key for navigation.  Visually the content is updated on the screen however due to the aria-hidden value set to true the updated content is not announced for screen reader users.  Recommend using valid markup and follow the rules of the ARIA specification as well as correctly setting ARIA properties, including roles based on the intended purpose and nest ARIA roles appropriately.  [More information about creating accessible carousels can be found on the W3C website](https://www.w3.org/WAI/tutorials/carousels/).

![aa401ec97dc8890d5f66da8bae72cbfb1ddc72a7-1641301381](https://user-images.githubusercontent.com/8166060/152891514-eb54ea96-023e-4331-b6b0-88d79c05c0c9.png)
